styles/modal.css

```
@@ -86,3 +86,88 @@ body.modal-open {
+
+  .modal-description h2 {
+    font-size: 32px;
+  }
+}
+
+@media (max-width: 390px) {
+  .modal {
+    top: 20px;
+    border-radius: 0;
+    bottom: 0;
+    width: 800px;
```

Stylelint (17.6.0)

[error] Unexpected duplicate "width" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)